pkg/services/ngalert/notifier/channels/googlechat.go

```diff
@@ -34,6 +35,8 @@ func NewGoogleChatNotifier(model *NotificationChannelConfig, t *template.Templat
 		return nil, receiverInitError{Cfg: *model, Reason: "could not find url property in settings"}
 	}
 
+	content := model.Settings.Get("message").MustString(`{{ template "default.message" . }}`)
```

Comment:
I would suggest moving the default template to a constant. that would benefit readability as well as ability to use it in tests

Reply:
Good idea. There are a number of channels that all reference this name, this is a good improvement across the board.